corehq/util/soft_assert/api.py

```diff
@@ -3,9 +3,18 @@
 from corehq.apps.hqwebapp.tasks import send_mail_async, mail_admins_async
 from corehq.util.global_request import get_request
 from corehq.util.soft_assert.core import SoftAssert
+from django.views.debug import get_exception_reporter_filter
 
 
 def _send_message(info, backend):
+    try:
+        request = get_request()
+        # de-sensitize requests that might have passwords
```

Comment:
what if we defined a `get_sanitized_request_repr` function that this could call and push all the logic there?